modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/ApplicationPolicyDeploymentTestCase.java

```diff
@@ -630,9 +630,11 @@ public void appliesPolicyUsingErrorTypeAndHavingDependencies() throws Exception
   @Test
   @Issue("MULE-18196")
   @Description("The application declares an ErrorType that is needed by the policy and but the policy doesn't have it in its own ErrorType repository")
-  public void appliesPolicyUsingErrorTypeDeclaredOnApp() throws Exception {
+  public void appliesPolicyUsingErrorTypeDeclaredOnAppDependency() throws Exception {
```

Comment:
What about `MULE-18196`? Does this failing test mean that we introduced a regression?

Reply:
that's what MULE-19203 is about